pandas/io/html.py

```diff
@@ -236,104 +238,104 @@ def _text_getter(self, obj):
         """
         raise AbstractMethodError(self)
 
-    def _parse_td(self, obj):
-        """Return the td elements from a row element.
+    def _equals_tag(self, obj, tag):
+        """Returns whether an individual DOM node matches a tag
 
         Parameters
         ----------
-        obj : node-like
+        obj : a DOM node.
+
+        tag : str
+            Tag to be checked for equality
 
         Returns
         -------
-        columns : list of node-like
-            These are the elements of each row, i.e., the columns.
+        is_tag_equal : boolean
+            boolean indicating if the object is equal to tag 'tag'
         """
         raise AbstractMethodError(self)
 
-    def _parse_tables(self, doc, match, attrs):
-        """Return all tables from the parsed DOM.
+    def _contains_tag(self, obj, tag):
+        """Returns whether an individual DOM node has a particular tag
+        contained within it
 
         Parameters
         ----------
-        doc : tree-like
-            The DOM from which to parse the table element.
+        obj : a DOM node.
 
-        match : str or regular expression
-            The text to search for in the DOM tree.
+        tag : str
+            Tag to be found in this DOM
 
-        attrs : dict
-            A dictionary of table attributes that can be used to disambiguate
-            mutliple tables on a page.
+        Returns
+        -------
+        boolean
```

Comment:
Similar to `is_tag_equal` above, provide a variable name here and then the data type.